src/ManticoreSearch/Client.php

```diff
@@ -89,16 +89,25 @@ public function setServerUrl($url): void {
 	 * @param string $request
 	 * @param ?string $path
 	 * @param bool $disableAgentHeader
+	 * @param bool $isRequestEncoded
 	 * @return Response
 	 */
-	public function sendRequest(string $request, string $path = null, bool $disableAgentHeader = false): Response {
+	public function sendRequest(
+		string $request,
+		string $path = null,
+		bool $disableAgentHeader = false,
+		bool $isRequestEncoded = true
+	): Response {
 		$t = microtime(true);
 		if (!isset($this->responseBuilder)) {
 			throw new RuntimeException("'responseBuilder' property of ManticoreHTTPClient class is not instantiated");
 		}
 		if ($request === '') {
 			throw new ManticoreSearchClientError('Empty request passed');
 		}
+		if ($isRequestEncoded === false) {
```

Comment:
When we actually may have already encoded? 
Instead of introducing a flag, maybe it's better to do URL encoding all the time and make sure that we have unencoded data as an argument. To keep it more simple.

Reply:
Now it's assumed in Manticore that all queries to the `/sql` endpoint must be encoded by user, otherwise user gets an error message. If Buddy implicitly does the 'user's job' here, it can make the whole matter a bit more vague.  At least, this should be covered in documentation then. Let's discuss it in 'team-buddy'.